peachjam/models/judgment.py

```diff
@@ -431,7 +432,7 @@ def pre_save(self):
 
         self.doc_type = "judgment"
         self.assign_mnc()
```

Comment:
I suggest putting an `if` before calling `assign_mnc` -- that's one of the key things that must be gated on the flag

Reply:
done